regression/cbmc-java/array_float_multidim_nonconstsize_nonconstaccess_2/test.desc

```diff
@@ -0,0 +1,6 @@
+CORE
+Test.class
+--function Test.f --cover location --unwind 3
+\d+ of \d+ covered
```

Comment:
Consider checking 100% rather than arbitrary coverage

Reply:
Not all of the tests that do not crash end up with 100% coverage, for various reasons. I can try to tweak them so that they do. That would definitely be helpful to make sure that we do not loose coverage on them when fixing the related bug.

Reply:
Constructors are bringing down the coverage even if the function under test is fully covered. CBMC does not have cover-function-only option so instead I added the lines with satisfied goals into .desc files.